scripts/depth/utils.py

```diff
@@ -0,0 +1,114 @@
+# Copyright Niantic 2019. Patent Pending. All rights reserved.
+#
+# This software is licensed under the terms of the Monodepth2 licence
+# which allows for non-commercial use only, the full terms of which are made
+# available in the LICENSE file.
+
+from __future__ import absolute_import, division, print_function
+import os
+import hashlib
+import zipfile
+from six.moves import urllib
+
+
+def readlines(filename):
+    """Read all the lines in a text file and return as a list
+    """
+    with open(filename, 'r') as f:
+        lines = f.read().splitlines()
+    return lines
+
+
+def normalize_image(x):
+    """Rescale image pixels to span range [0, 1]
+    """
+    ma = float(x.max().cpu().data)
+    mi = float(x.min().cpu().data)
+    d = ma - mi if ma != mi else 1e5
+    return (x - mi) / d
+
+
+def sec_to_hm(t):
+    """Convert time in seconds to time in hours, minutes and seconds
+    e.g. 10239 -> (2, 50, 39)
+    """
+    t = int(t)
+    s = t % 60
+    t //= 60
+    m = t % 60
+    t //= 60
+    return t, m, s
+
+
+def sec_to_hm_str(t):
+    """Convert time in seconds to a nice string
+    e.g. 10239 -> '02h50m39s'
+    """
+    h, m, s = sec_to_hm(t)
+    return "{:02d}h{:02d}m{:02d}s".format(h, m, s)
+
+
+def download_model_if_doesnt_exist(model_name):
+    """If pretrained kitti model doesn't exist, download and unzip it
```

Comment:
We need to delete this function, right? Because these are all pytorch models. Even users download it, they are not able to load it.